### DISCUSSION TITLE
epwalsh/obsidian.nvim

### DISCUSSION THREAD

**Discussion by tellcodes** [vscode-neovim/vscode-neovim#1634] [General]
Hi,

I tried this plugin: https://github.com/epwalsh/obsidian.nvim in my init.lua.

However when I run :ObsidianToday the file is opened in a read-only mode editor. Is there any way to open the file in a normal editor.

<img width="623" alt="image" src="https://github.com/vscode-neovim/vscode-neovim/assets/48771357/1ba0449b-2e53-4320-978a-247deb43aac1">

**Comment by xiyaowong** [comment]
This category of plugins is not supported

For this category of plugin, it's recommended to directly use Neovim. Using VSCode does not provide any additional benefits.

**Reply by tellcodes** [reply]
Ok, thanks for replying! 👍
